gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/TemporalGraphTest.java

```diff
@@ -193,4 +199,26 @@ public void testFromLogicalGraph() throws Exception {
     loadedVertices.forEach(this::checkDefaultTemporalElement);
     loadedEdges.forEach(this::checkDefaultTemporalElement);
   }
+
+  /**
+   * Test the {@link TemporalGraph#fromGraph} method with TimeInterval Extractors as parameters
+   *
+   * @throws Exception if loading the graph from the csv data source fails
+   */
+  @Test
+  public void testFromGraphWithTimeIntervalExtractors() throws Exception {
+
+    String path = getFilePath("/data/csv/socialnetwork/");
+    TemporalDataSource csvDataSource = new TemporalCSVDataSource(path, getConfig());
+    TemporalGraphCollection temporalGraphCollection = csvDataSource.getTemporalGraphCollection();
+    TemporalGraph expected = temporalGraphCollection.reduce(new ReduceCombination<>());
+    LogicalGraph logicalGraph =
```

Comment:
code style
```
    LogicalGraph logicalGraph = getTemporalSocialNetworkLoader().getGraphCollection()
      .reduce(new ReduceCombination<>());

    TemporalGraph check = TemporalGraph.fromGraph(logicalGraph, 
      g -> TemporalGradoopTestUtils.extractTime(g), 
      v -> TemporalGradoopTestUtils.extractTime(v), 
      e -> TemporalGradoopTestUtils.extractTime(e));
```

Reply:
yes, definetly looks better but get´s refused by the checkstyle plugin here in github:

[ERROR] /home/runner/work/gradoop/gradoop/gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/TemporalGraphCollectionTest.java:284: 'lambda arguments' have incorrect indentation level 8, expected level should be one of the following: 4, 6. [Indentation]